hyperapp.d.ts

```diff
@@ -13,12 +13,12 @@ export interface VNode<Attributes = {}> {
   key: string
 }
 
-/** A Component is a function that returns a custom VNode.
+/** A Component is a function that returns a custom VNode or View.
  *
  * @memberOf [VDOM]
  */
-export interface Component<Attributes = {}> {
-  (attributes: Attributes, children: Array<VNode | string>): VNode<Attributes>
+export interface Component<Attributes = {}, State = any, Actions = any> {
```

Comment:
Maybe we should prefer the use of `{}` instead of `any`, here. 
What do you think ?

Reply:
First I used `{}` then changed to `any` in order to avoid breaking change... but, wait, there isn't breaking change (because `View<State, Actions>` is added by myself). Get back to `{}`